doc/source/text.rst

```diff
@@ -306,21 +306,26 @@ The same alignment can be used when ``others`` is a ``DataFrame``:
 Concatenating a Series and many objects into a Series
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All one-dimensional list-likes can be combined in a list-like container (including iterators, ``dict``-views, etc.):
+Several items can be combined a list-like container (including iterators, ``dict``-views, etc.), which may contain ``Series``, ``Index`` and ``np.ndarray``.
+Note that ``Index`` will align as well, so we change the indexes of ``s`` and ``u`` to strings for the purpose of this example:
 
 .. ipython:: python
 
-    s
-    u
-    s.str.cat([u.values,
-               u.index.astype(str).values], na_rep='-')
+    s_values = np.array(['a', 'b', 'c', 'd'], dtype=object)  # same as s.values
```

Comment:
Hm, I though this was clearly referencing the actual series `s` (as I wanted to motivate the variable name), rather than make a general statement about the relationship between `np.array` and `.values`.

Reply:
i agree, the comment is just confusing